tests/pytests/test_geometry_flat.py

```diff
@@ -264,7 +264,7 @@ def testFtInfo(env):
   res = to_dict(env.cmd('FT.INFO idx1'))
   env.assertEqual(float(res[info_key_name]), cur_usage)
 
-  doc_num = 100
+  doc_num = 10000
```

Comment:
Is this enough to cause mem relocation? Can we somehow make sure it does?

Reply:
Experimentally. This value always caused a mem relocation related crash prior to this fix.

We should, however, find a way to force mem relocation regardless.